modules/core/metrics/metrics.go

```diff
@@ -1,7 +1,16 @@
-package types
+package metrics
 
 // Prometheus metric labels.
 const (
+	// 02-client labels
+
+	LabelClientType = "client_type"
```

Comment:
In the very strict sense this is API breaking, but I am not sure if it's worth documenting it in the migration docs. Maybe a changelog would be good, though?